assets/sass/components/idea-hub/_googlesitekit-idea-hub.scss

```diff
@@ -0,0 +1,63 @@
+/**
+ * Idea Hub DashboardCTA styles
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin {
+
+	._googlesitekit-idea-hub-dashboardcta {
+		background-color: $c-white;
+		display: flex;
+		flex-wrap: wrap;
+		justify-content: center;
+	}
+
+	._googlesitekit-idea-hub-dashboardcta__content {
+		display: inline-block;
+		max-width: 300px;
+		padding: $grid-gap-desktop;
+
+		h5 {
+			font-family: $f-secondary;
+			font-size: 22px;
+			font-weight: $fw-secondary-normal;
+			margin: 0;
+			text-align: left;
+		}
+	}
+
+	._googlesitekit-idea-hub-dashboardcta__icon {
+		display: inline-flex;
+		justify-content: center;
+		margin: $grid-gap-desktop 0 $grid-gap-desktop $grid-gap-desktop;
+		width: 144px;
+	}
+
+	._googlesitekit-idea-hub-dashboardcta__icon svg {
+		background-color: $c-idea-hub-orange;
+		background-size: 144px 144px;
+		border-radius: 50%;
+		height: 144px;
+		padding: 32px;
+		width: 144px;
+	}
+
+	._googlesitekit-idea-hub-dashboardcta__learnmore::before {
+		content: url("data:image/svg+xml,%3Csvg width='10' height='14' viewBox='0 0 10 14' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M3.00004 13C3.00004 13.3667 3.30004 13.6667 3.66671 13.6667H6.33337C6.70004 13.6667 7.00004 13.3667 7.00004 13V12.3334H3.00004V13ZM5.00004 0.333374C2.42671 0.333374 0.333374 2.42671 0.333374 5.00004C0.333374 6.58671 1.12671 7.98004 2.33337 8.82671V10.3334C2.33337 10.7 2.63337 11 3.00004 11H7.00004C7.36671 11 7.66671 10.7 7.66671 10.3334V8.82671C8.87337 7.98004 9.66671 6.58671 9.66671 5.00004C9.66671 2.42671 7.57337 0.333374 5.00004 0.333374ZM6.90004 7.73337L6.33337 8.13337V9.66671H3.66671V8.13337L3.10004 7.73337C2.65837 7.42718 2.29733 7.01867 2.04774 6.54271C1.79815 6.06675 1.66742 5.53747 1.66671 5.00004C1.66671 3.16004 3.16004 1.66671 5.00004 1.66671C6.84004 1.66671 8.33337 3.16004 8.33337 5.00004C8.33337 6.08671 7.80004 7.10671 6.90004 7.73337Z' fill='%231A73E8'/%3E%3C/svg%3E");
```

Comment:
Is there a reason we need to add the icon with CSS rather than in React as we normally would?

If we _really_ needed to, we could use `{ renderToString } from '@wordpress/element'` to get a string from a React element rather than duplicating it here.

Reply:
I was unsure of the convention here. Happy to move it over to React 👍

Reply:
Yeah, we should use svg components where possible as it allows much more flexibility, particularly around styling.